types/ember-data/v2/index.d.ts

```diff
@@ -1094,7 +1094,7 @@ export namespace DS {
          * This method unloads all records in the store.
          * It schedules unloading to happen during the next run loop.
          */
-        unloadAll<K extends keyof ModelRegistry>(modelName: K): void;
+        unloadAll<K extends keyof ModelRegistry>(modelName?: K): void;
```

Comment:
Please ensure that this change is made to the v3 types as well (including the same test)

Reply:
Thanks Mike. Where do I find these v3 types? I've made this change to two sets of types in this repo.

Reply:
aha. I must have missed the other file in the github phone ui